Libraries/Opc.Ua.Server/Diagnostics/MonitoredNode.cs

```diff
@@ -279,13 +279,31 @@ public void OnMonitoredNodeChanged(ISystemContext context, NodeState node, NodeS
 
                     if (monitoredItem.AttributeId == Attributes.Value && (changes & NodeStateChangeMasks.Value) != 0)
                     {
-                        QueueValue(context, node, monitoredItem);
+                        ServerSystemContext serverSystemContext;
```

Comment:
Couldn´t all of this code life directly in the QueueValue Method of the MonitoredNode to avoid duplicate code?

there is also a constructor for operationContext that directly takes a monitored Item, that seems more readable to me, this could be the resulting change needed in the QueueValue method:

```C#
context = (context as ServerSystemContext)?.Copy(new OperationContext(monitoredItem)) ?? context;
```
FIY: I did not yet test the change, just what if have noticed looking at the changes

Reply:
@Filippo-Oliva-ABB

Reply:
Hi @romanett, 

  Thank for your suggestion.
I agree with your point and have made changes to the OnMonitoredNodeChanged method to just to keep potential regressions low.